src/ProtocolGateway.Core/Instrumentation/MqttIotHubAdapterEventSource.cs

```diff
@@ -45,7 +45,7 @@ public bool IsErrorEnabled
         [Event(VerboseEventId, Level = EventLevel.Verbose)]
         public void Verbose(string message, string info)
         {
-            if (this.IsVerboseEnabled)
+//            if (this.IsVerboseEnabled)
```

Comment:
For some reason it may be disabled. The right fix is to either enable it explicitly or make sure that nobody disables it. I'm working on it